Add Velsko 2024

alexhbnr opened the pull request:
Using the tables provided by @ivelsko.

# Pull Request

This PR is for a

- [x] [New Publication(s)](#new-publication)
- [ ] [Correction](#correction)

For the following list(s):

- [ ] ancientmetagenome-environmental ([README](https://github.com/SPAAM-workshop/AncientMetagenomeDir/tree/master/ancientmetagenome-environmental))
- [x] ancientmetagenome-hostassociated ([README](https://github.com/SPAAM-workshop/AncientMetagenomeDir/tree/master/ancientmetagenome-hostassociated))
- [ ] ancientsinglegenome-hostassociated ([README](https://github.com/SPAAM-workshop/AncientMetagenomeDir/tree/master/ancientsinglegenome-hostassociated))

This is to close #1206.

### Pre-review checklist (new publications)

- [x] Publication is published
  - Preprints currently not accepted?
- [x] Checked the publication is not already in the database?
- [x] Checked samples in this publication are not previously published _data_?
  - Newly re-sequenced metagenomes are OK!
- [x] Samples are shotgun metagenomes and not amplicon data
  - Note: hostassociated-singlegenome may also contain whole-genome enriched data
- [x] Checked the list follows conventions as described in the corresponding sample type's README file (e.g. using ERS/SRS accession codes for ENA/SRA)?
- [x] **Once sample table validation completed** Library metadata has been added
  - - [x] Changelog is updated to include the publication under 'Added'?

jfy133 commented:
## General

I note the following missing samples from the Dir table:

- AMH003.A0101

I note the adding of additional samples not in the Supp data of the paper: 

- RN36

However I think these are the same, just there is a typo in teh supp. mat. table where arch. ID RN36 has been replaced with `D` for the library AMH003.A0101? Can you confirm @alexhbnr ?

Additionally some of the lat:lons are highly duplicated, is it correct they are from all the exact same place? e.g.,  Ahu Kihi Kihi Rau Mea, Ahu Mahatua, Ahu Nau Nua, Ahu Onero, Ahu Taurtira, Akahanga Karava, Ana Koe Hoko (I think these are almost all the Rapa Nui ones). If we don't have the exact location (or not allowed) we should maybe drop this to just two decimals. 

## Samples

- We could to add the island names after the site name (at least for the Rapa Nui - Chile is slightly misleading geographically), also 
- sample_name: 1/1 - should be skeletal tissue / bone
- sample_name(s): B3/B8: the Lat:Lon sends us to the middle of the Pacific ocean, but the Island column of the supp. data. syas' 'Watom Bismarck Archpeilgo' and also there is an island called Watom Island... maybe the lat:lon should fall on that?
- sample_names: BG3 to SK9_3, the Lat:Lon falls a 20m drive to the west of a google-maps hit called 'Talasiu' maybe the latter is the correct one?
-

alexhbnr commented:
@jfy133 regarding AMH003 vs. RN36, this is indeed the same sample according to our Pandora database.

Regarding the comments on the coordinates or site names:
- for the Rapa Nui sites, I have added "Rapa Nui" to the site names and have updated the coordinates to match the records in the Pandora database
- updated the coordinates of B3/B8 to match the Wahom site
- for the Tonga site, the coordinates in the table are matching the ones described in this article for the Talasiu site: https://pacificarchaeology.org/index.php/journal/article/view/92

Regarding the polymerase, I will leave it at the moment as is but might update it later once I got feedback from @ivelsko.
